src/components/FinalPoem.js

```diff
@@ -4,15 +4,24 @@ import './FinalPoem.css';
 
 const FinalPoem = (props) => {
 
+  const finalPoem = props.submissions.map((line, i) => {
+    return(
+      <li key={i}>
+        {line}
+      </li>
+    )
+  });
+
   return (
-    <div className="FinalPoem">
-      <section className="FinalPoem__poem">
+    <div className='FinalPoem'>
+      <section className={`FinalPoem__poem ${props.isSubmitted ? 'visible' : 'hidden'}`}>
```

Comment:
You're hiding the content based on CSS while the test just assumes it's not rendered at all, which is why the test fails.